VideoLocker/res/layout/row_discussion_comments_response.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.v7.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_marginLeft="@dimen/discussion_responses_standard_margin"
+    android:layout_marginRight="@dimen/discussion_responses_standard_margin"
+    android:layout_marginTop="@dimen/discussion_responses_standard_margin"
+    android:layout_marginBottom="@dimen/discussion_responses_standard_margin"
```

Comment:
If all the margins are identical, then you can simple set `android:layout_margin` to the desired value, instead of setting each individual `android:layout_margin*` property to the same value.